### ISSUE TITLE
Version is not target at version (automatic update)

### ISSUE THREAD

**Issue by daniel-lo** [TasmoAdmin/TasmoAdmin#1234]
I can't get automatic bulk update to work. It seems to start well but it then sais "Version is not at target version".

Currently trying to go from 14.1.0 to 14.2.0

via: ota.tasmota.com

[2024-08-20T10:09:36.317Z] Starte Update Prozess[2024-08-20T10:09:36.317Z] Attempting upgrade to version 14.2.0[2024-08-20T10:09:36.317Z] Checking version...[2024-08-20T10:09:36.623Z] Current version is 14.1.0(release-tasmota)[2024-08-20T10:09:36.623Z] OTA Url => [http://ota.tasmota.com:80/data/firmwares/tasmota.bin.gz[2024-08-20T10:09:36.975Z]](http://ota.tasmota.com/data/firmwares/tasmota.bin.gz%5B2024-08-20T10:09:36.975Z%5D) Starte Module update...[2024-08-20T10:09:37.285Z] Sleeping for 10s...[2024-08-20T10:09:47.297Z] Update gestartet[2024-08-20T10:09:47.297Z] Checking version...[2024-08-20T10:09:47.494Z] Version is not at target version[2024-08-20T10:09:47.494Z] Sleeping for 10s...[2024-08-20T10:09:57.499Z] Checking version...[2024-08-20T10:09:57.708Z] Version is not at target version[2024-08-20T10:09:57.708Z] Sleeping for 10s...

The top banner is green:
![grafik](https://github.com/user-attachments/assets/1836e1a6-508c-498b-81e2-ca7e8e02dbe0)

tasmoadmin V4.1.2

If I enter this url in the device's OTA updgrade field it works:
http://ota.tasmota.com/tasmota/release/tasmota.bin.gz

So I suppose the appended part of the URL needs to be fixed. Or nothing should be appended and the same link should be valid that works via device GUI.

**Comment by daniel-lo** [comment]
I tried to use localhost (Docker version) instead of ota.tasmota.com and the exact same thing happens.

**Comment by daniel-lo** [comment]
Yes .. still relevant. I can't do anything about it since I am not a programmer.

**Comment by inverse** [comment]
The OTA server IP  that's given to the devices should be the IP address (or host) of the device running TasmotaAdmin.

**Comment by inverse** [comment]
![image](https://github.com/user-attachments/assets/ccbe44c7-3985-4ef7-9747-b29d66d14cd1)

For example this is mine as It's running on `192.168.1.100`

**Comment by daniel-lo** [comment]
@inverse 

I had tried that but not with 9541 as a port.  Now that I use the same port I get:

Minimal Firmware is not accessible on the server at http://192.168.178.118:9541/data/firmwares/tasmota-minimal.bin.gz
Please check the OTA IP address and port are correct
Full Firmware is not accessible on the server at http://192.168.178.118:9541/data/firmwares/tasmota.bin.gz
Please check the OTA IP address and port are correct

**Comment by daniel-lo** [comment]
I must add I am using the docker version which lags a bit behind the releases.

**Comment by inverse** [comment]
@daniel-lo That port is the port I am exposing TA on. What is the port you are using for your docker setup? How are you running things?

**Comment by inverse** [comment]
TA works by pulling the upstream OTA files locally so your LAN devices can fetch from your LAN and not go out to the internet.

**Comment by daniel-lo** [comment]
@inverse 

Thanks .. that helped.

I am not a sophisticated docker user - the opposite is actually the case. So I manually run this after each boot:

docker run -it -p 8282:80 -v ~/tasmo_data:/data ghcr.io/tasmoadmin/tasmoadmin:latest

Using port 8282 with the local IP address seems to solve it.
Regrettably all devices are currently on the newest build so I can't make the final test.

**Comment by charettepa** [comment]
I am still having the same issue as OP
Its been present for almost a year
I upgrade everytime there is a new version

If i try to access the exact address its going to
http://192.168.0.15:9541/data/firmwares/tasmota-minimal.bin.gz
I get an nginx error 403 Forbiden
